skimage/measure/_regionprops.py

```diff
@@ -346,6 +364,14 @@ def __getattr__(self, attr):
                     f'be 1 or 2, but {attr} takes {n_args} arguments.'
                 )
         elif attr in PROPS and attr.lower() == attr:
+            if (
+                self._intensity_image is None
+                and PROPS[attr] in _require_intensity_image
+            ):
+                raise AttributeError(
```

Comment:
Is AttributeError is correct here; that means "attribute not found". Maybe RuntimeError? Not sure, what do you think?

Reply:
yeah, RuntimeError may be better. I will change it to that

Reply:
I take this back, I ran into several other test errors in other places when it is not an attribute error. All errors in the `__getattr__` method need to be of type `AttributeError` to meet other assumptions made elsewhere in `_regionprops.py`.

Reply:
OK, cool